docs/geedocs/5.2.2/answer/7160002.html

```diff
@@ -51,9 +58,37 @@ <h5>
       <li>Ubuntu 14.04 LTS and 16.04 LTS</li>
     </ul>
     <p>Google Earth Enterprise 5.2.2 is compatible with Google Earth Enterprise Client (EC) version 7.1.5 and above.</p>
-<br/>
-  <p>If you're upgrading from GEE 5.1.3, you may have to reinstall the prerequisite libraries after uninstalling 5.1.3 but before installing 5.2.1.</p>
+  <p><p>To upgrade from Open GEE 5.2.0, do NOT uninstall it. We recommend that you upgrade Open GEE 5.2.0 by simply installing Open GEE 5.2.2. Installing Open GEE 5.2.2 on top of Open GEE 5.2.0 will ensure that your PostgreSQL databases are backed-up and upgraded correctly to the new PostgreSQL version used by Open GEE 5.2.2.</p></p>
```

Comment:
For RPMs would you use -U instead of -i when 'installing over top' of the current install?  Also for YUM command yo want to use 'update' vs 'install'